integrations/docker/images/chip-build-telink/Dockerfile

```diff
@@ -1,51 +1,31 @@
 ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 
-# Telink Toolchain
-ARG TELINK_DIR=/opt/telink
-RUN dpkg --add-architecture i386 \
-    && apt-get update \
-    && apt-get install -y \
-    libc6:i386 \
-    libncurses5:i386 \
-    libstdc++6:i386 \
-    && mkdir ${TELINK_DIR} \
-    && cd ${TELINK_DIR} \
-    && wget http://wiki.telink-semi.cn/tools_and_sdk/Tools/IDE/telink_riscv_linux_toolchain.zip \
-    && unzip ${TELINK_DIR}/telink_riscv_linux_toolchain.zip -d ${TELINK_DIR} \
-    && rm ${TELINK_DIR}/telink_riscv_linux_toolchain.zip
+# Setup toolchain
+ENV ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+ENV ZEPHYR_SDK_INSTALL_DIR=/opt/telink/zephyr-sdk-0.13.0
+RUN wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.13.0/zephyr-toolchain-riscv64-0.13.0-linux-x86_64-setup.run\
+    && chmod +x zephyr-toolchain-riscv64-0.13.0-linux-x86_64-setup.run \
+    && ./zephyr-toolchain-riscv64-0.13.0-linux-x86_64-setup.run -- -d ${ZEPHYR_SDK_INSTALL_DIR} \
+    && rm zephyr-toolchain-riscv64-0.13.0-linux-x86_64-setup.run
 
 # Setup Zephyr
-ARG ZEPHYR_PROJECT_DIR=/opt/zephyrproject
+ARG ZEPHYR_PROJECT_DIR=/opt/telink/zephyrproject
 ENV PATH="/root/.local/bin:${PATH}"
-RUN DEBIAN_FRONTEND=noninteractive apt install -y \
-    git \
-    gperf \
+RUN apt update \
+    && DEBIAN_FRONTEND=noninteractive apt install -y \
     ccache \
     dfu-util \
     device-tree-compiler \
-    wget \
-    xz-utils \
     gcc-multilib \
     g++-multilib \
     libsdl2-dev \
     && pip3 install --user -U west \
     && pip3 install pyelftools \
-    && west init --mr=zephyr-v2.5.0 ${ZEPHYR_PROJECT_DIR} \
+    # @todo: Add --mr=zephyr-v2.7.0 as soon as this version would be released
```

Comment:
does this comment work? I am assuming '\' is "continue on the next line" however here:
 - this line is missing the \
 - it \ combines the entire lines, this  will comment out the entire followup

Reply:
Guessing this comment is a 'dockerfile comment' so it gets completely removed, however I wonder if this would be clearer if dockerfile comments would not be inline with what looks like bash code.